core-rust/core-api-server/src/core_api/handlers/lts/transaction_status.rs

```diff
@@ -106,6 +106,7 @@ fn handle_lts_transaction_status_internal(
         return Ok(models::LtsTransactionStatusResponse {
             intent_status,
             status_description: format!("The transaction has been committed to the ledger, with an outcome of {outcome}. For more information, use the /transaction/receipt endpoint."),
+            committed_state_version: Some(to_api_state_version(txn_state_version)?),
```

Comment:
I see this added in the LTS version only - intended?

Reply:
Yes - in the sense that it's the only one critical for the RCNet release, and the other one can follow later.